packages/rum-core/src/common/patching/event-target-patch.js

```diff
@@ -131,6 +133,13 @@ export function patchEventTarget(callback) {
     existingTasks.push(task)
 
     function wrappingFn() {
+      // To identify properly the element on which the event occurred we need to make sure we distinguish between currentTarget and target
+      // currentTarget always refers to the element to which the event handler has been attached
+      // target identifies the element on which the element occurred
+      const [event] = arguments
+      task.currentTarget = event.currentTarget
+      task.target = event.target
```

Comment:
In order to make the change more visible I added the currentTarget too as a property of our task. But I understand that we can remove it if we are not going to use it

What do you think?

Reply:
We can remove the currentTarget, the comment conveys it clearly. Thanks